spec/features/users/postal_address_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe 'Add Postal Address', type: :feature do
+  let(:user) { create(:user) }
+  let(:address) { create(:postal_address) }
```

Comment:
As said in a previous comment, I'd personally prefer to just use actual strings, like "Canada" in the `fill_in` bits, rather then an address build via a factory... Nevertheless, using `create` here is not what we want. We don't want to create an address in the database **BEFORE** the test -> it's more the outcome of the test 😉